### ISSUE TITLE
ACCESSIBILITY and STYLES are not applied when list items are created dynamically inside popper

### ISSUE THREAD

**Issue by dostarora97** [SAP/fundamental-ngx#8947]
#### Is this a bug, enhancement, or feature request?
Bug
#### Briefly describe your proposal.
Accessibility and styles are should be applied when list items are created dynamically inside popper
#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
- `"@fundamental-ngx/core": "0.36.0"`
- `"@angular/core": "14.0.2",`

#### If this is a bug, please provide steps for reproducing it.
- Focus on Popover trigger
- Press Enter
- Popover opens. Focus is on List header inside popover
- _But on pressing tab, focus does not move to next list item. And style is not correct_

https://user-images.githubusercontent.com/31973060/200335175-4632874b-1b08-4980-a5fc-25881be85222.mov

```HTML
<button fd-button [fdPopoverTrigger]="popover2">
  Template Popover Trigger 2
</button>

<fd-popover #popover2 [focusAutoCapture]="true" [focusTrapped]="true">
  <ul fd-list [noBorder]="true">
    <li fd-list-group-header>
      <span fd-list-title>List 2</span>
    </li>
    <ng-container *ngFor="let documentMenuItem of documentMenuItems">
      <ng-container
        [ngTemplateOutlet]="documentMenuItemTemplate"
        [ngTemplateOutletContext]="{ $implicit: documentMenuItem }"
      ></ng-container>
    </ng-container>
  </ul>
</fd-popover>

<ng-template #documentMenuItemTemplate let-documentMenuItem>
  <ng-container
    *ngIf="!documentMenuItem?.disabled; else documentMenuLinkItem_label"
  >
    <li fd-list-item class="document-menu-item">
      <a fd-list-link [href]="documentMenuItem.url">
        <span fd-list-title>
          {{ documentMenuItem.label }}
        </span>
      </a>
    </li>
  </ng-container>
  <ng-template #documentMenuLinkItem_label>
    <li fd-list-item class="document-menu-item is-disabled">
      <span fd-list-title class="is-disabled">
        {{ documentMenuItem.label }}
      </span>
    </li>
  </ng-template>
</ng-template>
```
#### Please provide relevant source code if applicable.

Stackblitz:
https://stackblitz.com/edit/document-menu-accessibility?file=src/app/popover-trigger-example.component.html

**Comment by shrvr** [comment]
Hello @dostarora97 
Thanks for raising issue.

To solve this, keep your template (here `#documentMenuItemTemplate`) within the `<ul fd-list>`. So, your code for `popover2 ` would be like this.

```html
<button fd-button [fdPopoverTrigger]="popover2">Template Popover Trigger 2</button>

<fd-popover #popover2 [focusAutoCapture]="true" [focusTrapped]="true">
    <ul fd-list [noBorder]="true">
        <li fd-list-group-header>
            <span fd-list-title>List 2</span>
        </li>
        <ng-container *ngFor="let documentMenuItem of documentMenuItems">
            <ng-container
                [ngTemplateOutlet]="documentMenuItemTemplate"
                [ngTemplateOutletContext]="{ $implicit: documentMenuItem }"
            ></ng-container>
        </ng-container>

        <ng-template #documentMenuItemTemplate let-documentMenuItem>
            <ng-container *ngIf="!documentMenuItem?.disabled; else documentMenuLinkItem_label">
                <li fd-list-item class="document-menu-item">
                    <a fd-list-link [href]="documentMenuItem.url">
                        <span fd-list-title>
                            {{ documentMenuItem.label }}
                        </span>
                    </a>
                </li>
            </ng-container>
            <ng-template #documentMenuLinkItem_label>
                <li fd-list-item class="document-menu-item is-disabled">
                    <span fd-list-title class="is-disabled">
                        {{ documentMenuItem.label }}
                    </span>
                </li>
            </ng-template>
        </ng-template>
    </ul>
</fd-popover>
```
